cohort-parent/pom.xml

```diff
@@ -1037,16 +1037,19 @@
 							<failOnWarning>true</failOnWarning>
 							<ignoredUsedUndeclaredDependencies>
 								<!-- Excluding this because the JAXB classes are provided by the JRE as of Java 8, but there are potentially other classes in jakarta that we don't want to wholesale exclude -->
-								jakarta.xml.bind:jakarta.xml.bind-api
+								<ignoredUsedUndeclaredDependency>jakarta.xml.bind:jakarta.xml.bind-api</ignoredUsedUndeclaredDependency>
+								<ignoredUsedUndeclaredDependency>com.sun.activation:jakarta.activation</ignoredUsedUndeclaredDependency>
```

Comment:
In my PR (#178), I did a bunch of work to try to normalize our javax.activation dependency. Because of different artifactId and groupId in transitive dependencies, we were actually pulling in several copies of the activation API and implementation JARs. I normalized onto the following....

```xml
		<dependency>
			<groupId>jakarta.activation</groupId>
		    	<artifactId>jakarta.activation-api</artifactId>
		    	<version>1.2.2</version>
		</dependency>
```